dbt/seeds/etl_full_row_counts.csv

```diff
@@ -299,7 +299,7 @@ _out_eia__yearly_heat_rate_by_unit,2020,2010
 _out_eia__yearly_heat_rate_by_unit,2021,1954
 _out_eia__yearly_heat_rate_by_unit,2022,1886
 _out_eia__yearly_heat_rate_by_unit,2023,1865
-_out_ferc1__yearly_plants_utilities,,7887
+_out_ferc1__yearly_plants_utilities,,8101
```

Comment:
Some non EIA tables were changed after the inclusion of the #4153 row counts test update. Now it checks for values in the spreadsheet where partition = NULL where it didn't before. These are all tables that slipped through the cracks because they weren't getting checked.

Reply:
did they not get changed with that PR??

Reply:
nope

Reply:
That does seem fishy.

Reply:
One way to check is to run the row count checks on `main`...I'll do that now

Reply:
Ok, the tests pass on `main` and on this branch, which doesn't make sense because I haven't updated my pudl db.

Reply:
okay... these guys were my main concern in this for sure (plus the 714 2024 line). i agree that it seems like these are straggler guys that originate from that pr.

Reply:
I mentioned this down below, but one of the reasons these didnt get updated is that the `dbt_helper` function doesn't auto-update any of the rows with NA partition, so I had to do all these by hand.